packages/browser/jest-dom.d.ts

```diff
@@ -1,6 +1,7 @@
 // Disable automatic exports.
 
 import { ARIARole } from './aria-role.ts'
+import { ComparatorRegistry, ScreenshotMatcherOptions } from './context.js'
```

Comment:
I know this creates a circular dependency between this file and `context.d.ts`, was wondering if it's fine tho.

To fix it I would have to move `ScreenshotOptions` out of `context.d.ts` and, as it's using `Locator`, at least half of the other types declared in it 😅

Reply:
If it works, it works 😄